spec/raven/instance_spec.cr

```diff
@@ -16,12 +17,10 @@ private class InstanceTest < Raven::Instance
   end
 end
 
-private class LoggerTest < Raven::Logger; end
-
 def build_instance_configuration
   Raven::Configuration.new.tap do |config|
     config.dsn = "dummy://12345:67890@sentry.localdomain:3000/sentry/42"
-    config.logger = LoggerTest.new(Log::MemoryBackend.new, Log::Severity::Info)
+    config.logger = Log.for(Raven::Logger::PROGNAME)
```

Comment:
Using the Crystal logger here to be able to use `Log.capture`. I'm not sure why, but when I initialize `Raven::Logger` here, the messages are coming through, even if it's initialized in the same way (e.g. `Raven::Logger.for("sentry")`. I'm also wondering if the `Raven::Logger` wrapper is even necessary.

Reply:
Ok, so it's `Raven::Logger.capture(builder: Raven::Logger.builder)`. Good to know for future reference. :)